### PR TITLE
Dotnet add authorization

### LINKED ISSUE #158: Add RBAC (authorization)

### PR DESCRIPTION
**Pull request by GodVenn**
Closes #158 
With this Authorization scheme, the user will need to have the role "Role.User", which is managed here:
https://portal.azure.com/#blade/Microsoft_AAD_IAM/ManagedAppMenuBlade/Users/objectId/8256d35d-5c07-440b-9f47-4abce94c8565/appId/ea4c7b92-47b3-45fb-bd25-a8070f0c495c

### CODE REVIEW

File: backend/api/appsettings.Development.json [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
       "Default": "Information",
-      "Microsoft.AspNetCore": "Warning"
+      "Microsoft": "Information",
+      "Microsoft.Hosting.Lifetime": "Information"
     }
-  }
-}
+  },
+  "AllowedHosts": "*"
+}
```
Comment (Christdej) [inline backend/api/appsettings.Development.json:14]: Add newline at end of file

File: backend/backend.sln [hunk 2]
```diff
@@ -0,0 +1,25 @@
+﻿
```
Comment (Christdej) [inline backend/backend.sln:1]: Do we want this file?

**Review by Christdej — CHANGES_REQUESTED** [review]

File: backend/backend.sln (hunk 2 above)
Comment (GodVenn) [reply backend/backend.sln:1]: Yes, I think so, I'm using visual studio, and if anyone else wants to to the same this file ties it all together

File: backend/api/appsettings.Development.json (hunk 1 above)
Comment (GodVenn) [reply backend/api/appsettings.Development.json]: Fixed

File: backend/backend.sln (hunk 2 above)
Comment (Christdej) [reply backend/backend.sln:1]: Sure

**Review by Christdej — APPROVED** [review]

**Pull request merged**